content/patterns/radio/examples/radio-activedescendant.html

```diff
@@ -25,129 +25,129 @@
-            <li id="rb22" role="radio" aria-checked="false">Home Delivery</li>
-            <li id="rb23" role="radio" aria-checked="false">Dine in Restaurant</li>
+            <li id="rb21" role="radio" aria-checked="false">受け取り</li>
+            <li id="rb22" role="radio" aria-checked="false">宅配</li>
+            <li id="rb23" role="radio" aria-checked="false">レストランで食事</li>
           </ul>
         </div>
-        <div role="separator" id="ex_end_sep" aria-labelledby="ex_end_sep ex_label" aria-label="End of"></div>
+        <div role="separator" id="ex_end_sep" aria-labelledby="ex_label ex_end_sep" aria-label="の終了"></div>
       </section>
 
       <section>
-        <h2>Accessibility Features</h2>
+        <h2>アクセシビリティ機能</h2>
         <ul>
           <li>
-            The radio button referenced by <code>aria-activedescendant</code> is scrolled into view when it is not visible in the graphical rendering.
-            This can occur under many conditions, but is most common when people with visual impairments use a browser's zoom feature to increase the size of content.
+            <code>aria-activedescendant</code> によって参照されるラジオボタンは、グラフィカルなレンダリングで表示されていない場合にスクロールして表示されます。
+            これは多くの状況で発生しますが、視覚障害のある利用者がブラウザのズーム機能を使用してコンテンツのサイズを拡大する場合に最も一般的です。
           </li>
-          <li>Uses CSS attribute selectors for synchronizing <code>aria-checked</code> state with the visual state indicator.</li>
+          <li>CSS 属性セレクタを使用して、<code>aria-checked</code> 状態を視覚的な状態インジケータと同期します。</li>
           <li>
-            Uses CSS <code>:hover</code> and <code>:focus</code> pseudo-classes for styling visual keyboard focus and hover.
+            視覚的なキーボードフォーカスとホバーのスタイル設定には、CSS の <code>:hover</code> 及び <code>:focus</code> 擬似クラスを使用します。
             <ul>
-              <li>The focus indicator encompasses both the radio button and label, making it easier to perceive which option is being chosen.</li>
-              <li>Hover changes background of both the radio button and label, making it easier to perceive that clicking either the label or button will activate the radio button.</li>
-              <li>The cursor is changed to a pointer when hovering over the radio button to help people with visual impairments identify it as an interactive element.</li>
+              <li>フォーカスインジケータはラジオボタンとラベルの両方を囲み、どのオプションが選択されているかを認識しやすくします。</li>
+              <li>ホバーするとラジオボタンとラベルの両方の背景が変化し、ラベル又はボタンのいずれかをクリックするとラジオボタンがアクティブになることを認識しやすくします。</li>
+              <li>視覚障害のある利用者がインタラクティブな要素として識別できるように、ラジオボタンにカーソルを合わせるとカーソルがポインタに変わります。</li>
             </ul>
           </li>
           <li>
-            Because transparent borders are visible on some systems with high contrast enabled, only the focused element has a visible border.
-            When focusable elements are not focused, they have a 0-width border and additional padding equal in width to the border that is used to indicate focus.
+            ハイコントラストが有効になっている一部のシステムでは透明な境界線が表示されるため、フォーカスされた要素のみが可視境界線を持ちます。
```

Comment:
単純な表示の意味ではなく、「普段見えないものが見えるようになる」という意味なので、「可視化」くらいですかね？
```suggestion
            ハイコントラストが有効になっている一部のシステムでは透明な境界線が可視化されるため、フォーカスされた要素のみが可視境界線を持ちます。
```
この文は結構わかりづらいから訳注あってもいいかもですね。